CHANGELOG.md

```diff
@@ -1,5 +1,68 @@
 # Changelog
 
+## [Unreleased](https://github.com/ministryofjustice/hmpps-book-secure-move-api/tree/HEAD)
```

Comment:
I guess this will go away when we cut the new tag?

Reply:
@cwrw  hmm, good question, let me regenerate this...